lucet-runtime/tests/instruction_counting.rs

```diff
@@ -53,9 +90,12 @@ pub fn check_instruction_counts() {
         inst.run("test_function", &[]).expect("instance runs");
 
         let instruction_count = inst.get_instruction_count();
+        if let None = instruction_count {
+            panic!("instruction count expected from instance");
```

Comment:
just a style thing: this `if` could be folded in with the `unwap` below into `instruction_count.expect("has instruction counting")`

Reply:
Hm.  I attempted something like that but got a compiler message like "`expect` not available for u64`.  Possible problem between me and the keyboard.  I may be missing something.